buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -184,23 +184,6 @@ let docker_step
                   ]
                 , TestExecutive = [] : List DockerImage.ReleaseSpec.Type
                 , LogProc = [] : List DockerImage.ReleaseSpec.Type
-                , Toolchain =
```

Comment:
Toolchain is built always from mina-toolchain pipeline so it should not be built from other CI pipelines